script.100fm/resources/settings.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<settings>
+    <category>
+        <setting label="10000" type="number" id="channel" default="0" visible="false"/>
```

Comment:
Btw, this was really the last issue I wanted to point out.. the rest looks good.